talend-component-maven-plugin/src/it/web/.jenkins/Jenkinsfile

```diff
@@ -112,9 +73,8 @@ pipeline {
    */
   agent {
     kubernetes {
-      label _POD_LABEL
-      yaml _POD_CONFIGURATION
-      defaultContainer _STAGE_DEFAULT_CONTAINER
+      yamlFile 'talend-component-maven-plugin/src/it/web/.jenkins/jenkins_pod_api_test.yml'
```

Comment:
Because of the download folder that doesn't exist on the normal build it is not the same pod definition.
` { name: efs-jenkins-component-runtime-download, mountPath: /tmp/download }`
The other pod definition is here:  
https://github.com/Talend/component-runtime/blob/master/.jenkins/jenkins_pod.yml